content/en/docs/contribute/style/hugo-shortcodes/index.md

```diff
@@ -95,14 +95,30 @@ using the online documentation, the glossary entry displays a tooltip.
 
 The raw data for glossary terms is stored at [https://github.com/kubernetes/website/tree/master/content/en/docs/reference/glossary](https://github.com/kubernetes/website/tree/master/content/en/docs/reference/glossary), with a content file for each glossary term.
 
-### Glossary Demo
+### Glossary demo
 
 For example, the following include within the markdown renders to {{< glossary_tooltip text="cluster" term_id="cluster" >}} with a tooltip:
 
-```liquid
+```
 {{</* glossary_tooltip text="cluster" term_id="cluster" */>}}
 ```
 
+As well as tooltips, you can reuse the definitions from the glossary in page content.  
+Here's a short glossary definition:
+
+```
+{{</* glossary_definition prepend="A cluster is" term_id="cluster" length="short" */>}}
+```
+which renders as:
+{{< glossary_definition prepend="A cluster is" term_id="cluster" length="short" >}}</i>
```

Comment:
@sftim, Why have you included `</i>`?

Reply:
Ah, an error!

Reply:
Oops. That was left over from an idea I didn't use.